api/camunda/camunda-openapi.yaml

```diff
@@ -949,6 +949,7 @@ paths:
         :::note
         This endpoint is an [alpha feature](/components/early-access/alpha/alpha-features.md) and may be subject to change
         in future releases.
+        :::
```

Comment:
Nice, I think what happened here is that (1) the previous version had the note added by hand, and missed the closing tag...and then (2) this version, which had the note added by the generation process, filled it in correctly.

Reply:
We should still fix this upstream, though 😄